templates/repo/header.tmpl

```diff
@@ -100,7 +100,7 @@
 							{{else if and (not $.CanSignedUserFork) (eq (len $.UserAndOrgForks) 0)}}
 								data-content="{{$.locale.Tr "repo.fork_from_self"}}"
 							{{end}}
-						data-position="top center" tabindex="0">
```

Comment:
Hmm, why were they added in the first place, if they are now removed again? :thinking:

Reply:
Okay, according to the blame that's been that way since Gogs, I stopped looking after something like 7 years ago, where it was already present…

Reply:
> Hmm, why were they added in the first place, if they are now removed again? thinking

I don't know why they were added, but now are redundant because the focus should be in the button only.